docs/setup-guide/arktos-with-mizar-cni.md

```diff
@@ -0,0 +1,45 @@
+# Arktos deployment with Mizar CNI 
+
+The machines in this lab used are AWS EC2 t2-large (2 CPUs, 8GB mem), Ubuntu 18.04 LTS.
+
+The steps might change with the progress of development.
+
+1. To install dependencies required for Arktos, run the following commands: 
+```bash
+wget https://raw.githubusercontent.com/CentaurusInfra/arktos/master/hack/setup-dev-node.sh
+sudo bash setup-dev-node.sh
+git clone https://github.com/CentaurusInfra/arktos.git ~/go/src/k8s.io/arktos
+echo export PATH=$PATH:/usr/local/go/bin\ >> ~/.profile
+echo cd \$HOME/go/src/k8s.io/arktos >> ~/.profile
+source ~/.profile
+```
+  
+2. To check kernel, run the following command:
+
+```bash
+uname -a
+```
+
+If kernel version is `5.6.0-rc2` or higher then you can skip the step 3.
+
+3. To update Kernel, run following commands:
```

Comment:
Kernel with small k. You know the English language?

Reply:
Change the wording to "Update the kernel if kernel version is below `5.6.0-rc2`"
This should still be under step 1.